sdk/src/main/java/io/opentelemetry/sdk/trace/SpanData.java

```diff
@@ -172,6 +173,23 @@
    */
   public abstract boolean getHasRemoteParent();
 
+  /**
+   * Returns whether this Span has already been ended.
+   *
+   * @return {@code true} if the span has already been ended, {@code false} if not.
+   * @since 0.4.0
+   */
+  public abstract boolean getHasEnded();
```

Comment:
I recommend just "hasEnded" without the extraneous `get`

Reply:
Unfortunately, AutoValue seems to require a `get`-prefix here (see also `getHasRemoteParent`).

Reply:
yet another reason for me to hate autovalue

Reply:
I'm trying to decide if `isEnded` is better, or just as ugly

Reply:
`isFinished` ?

Reply:
I think `isFinished` sounds like it would be true if the span is exported (finished passing through the span pipeline). Maybe a name for the inverse could be used, like `isRunning`.

Reply:
I like that idea!

Reply:
`isActive`? just to throw out another option

Reply:
Being "active" (on a scope) is already another concept in OpenTelemetry, unfortunately.

Reply:
I like `isRunning`